src/yb/master/backfill_index.cc

```diff
@@ -678,6 +667,21 @@ BackfillTable::BackfillTable(
   done_.store(false, std::memory_order_release);
 }
 
+void BackfillTable::retrieve_index_names(const std::unordered_set<std::string>& requested_index_ids, std::string& requested_id_names) {
+  std::ostringstream out;
+  out << "{ ";
+  bool first = true;
+  for (const auto& index_id : requested_index_ids) {
+    if (!first) {
+      out << ", ";
+    }
+    out << master_->catalog_manager()->GetTableInfo(index_id)->name();
```

Comment:
Probably a pre-existing bug here, but it may be better to check/skip in case `master_->catalog_manager()->GetTableInfo(index_id)` returns nullptr